mlir/test/e2e/conv2d_wrw_perf_config.cfg

```diff
@@ -0,0 +1,2 @@
+if not config.allow_e2e_tests or config.no_AMD_GPU or not config.arch_support_mfma:
```

Comment:
Is `config.unsupported` inherited from parent directories? That is, do you need those first two terms in this `if`?